frontend/packages/dev-console/integration-tests/features/Helm/default-HelmCharts-Installation.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Helm Chart
+    User will be able to install the helm charts
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: The Helm Chart option on the +Add Page: HR-01-TC01
```

Comment:
Scenario and outcome doesn't match,  please check and update the scenario title appropriately